src/prisma/schema.prisma

```diff
@@ -3298,6 +3298,12 @@ model ScoutWallet {
   @@index([primary, scoutId])
 }
 
+model WalletSanctionStatus {
```

Comment:
i think this should just live on the wallet table. there's no one-to-many relationship or anything. and you can just add a single "sanctionedAt" column to capture the time and state